tests/subtractions/__snapshots__/test_api.ambr

```diff
@@ -34,7 +34,7 @@
           'name': 'palm.fa.gz',
         },
       ],
-      'subtraction_id': 'abc123',
+      'subtraction_id': None,
```

Comment:
Also seems wrong, nothing about the intended changes should result in the ID changing